plugin/rpcplugin/supervisor.go

```diff
@@ -0,0 +1,28 @@
+package rpcplugin
+
+import (
+	"github.com/mattermost/platform/plugin"
+	"github.com/mattermost/platform/plugin/pluginenv"
+)
+
+// Supervisor implements a plugin.Supervisor that launches the plugin in a separate process and
+// communicates via RPCs.
```

Comment:
This will be setting up the RPC wrappers and actually launching the process. The bulk of the remaining work will be in this package ironing that out.